### ISSUE TITLE
some basic question about using remora

### ISSUE THREAD

**Issue by charliechen912ilovbash** [nanoporetech/remora#28]
Hi,

Thanks for developing this useful tool.
I am current using 
(1). `Megalodon` + `Remora` 
(2). `Bonito` + `Remora`
for 5mc and 5hmc detection in human genome.

May I ask some basic questions about running this pipeline:
1. what is the difference between `sssI_fast5s`  and `pcr_fast5s` in the example ?
2. Just want to confirm if I understood correctly about what each command do:
`megalodon` or `bonito basecaller`: annotate the modified bases in fast5 files
`python  taiyaki/misc/merge_mappedsignalfiles.py`: map the modified base annotations to reads 
`remora  dataset prepare`: use the mapped annotations to prepare data for remora model training
`remora  model train`: train remora model 
`remora  infer`: use the trained model to infer modified bases. 

and also, is above the most recommended way for modified base detection ? 
I mean, does "`bonito` + `remora`" has better accuracy than using `bonito` alone with pre-trained `remora` model ?
(or, does "`megalodon` + `remora`" has better accuracy than using `megalodon` alone with pre-trained `remora` model ?) 

3. in the first step, `megalodon` and `bonito` which is more recommended ? (i.e. high accuracy) 

Thank you very much.

**Comment by cjw85** [comment]
Hi @charliechen912ilovbash,

If you are solely interested in calling modified bases, then you should use the Guppy basecaller. This can perform 5mC and 5hmC basecalling, and is fully supported by our TS teams.

You should only need to use remora if you are interested in training new basecallers to detect additional modified bases.

**Comment by charliechen912ilovbash** [comment]
@cjw85 
Thanks for replying. I see, I think I misunderstood what remora does.
So compared to megalodon and bonito, Guppy has better performance on 5mC and 5hmC detection?
Also, the 'additional modified bases' you mentioned, does it mean like 5mA or some other types of modification?

**Comment by marcus1487** [comment]
The accuracy should be equal using Guppy now (as of version 6.1.1) and the compute performance should be much better and is supported (not a research product). 

I'm not aware of 5mA, but 6mA or 4mC would be examples of modified bases existing in native samples for which we do not currently have a released model. The major use case for Remora at the moment would be to allow the community to train such models. 

To answer your specific questions:
> what is the difference between sssI_fast5s and pcr_fast5s in the example ?
sssI is a commercially available bacterial methylase which convert cytosine in CG sequence contexts to 5mC. PCR is polymerase chain reaction (in other words a sample with all modified bases removed via a PCR reaction). The sssI enzyme is applied to this PCR sample to create the training data for our production 5mC CG-context model. 

For the second question, bonito is not mentioned in the training pipeline (at the moment, but will be involved in the next relesase). Bonito is currently only used at inference time to apply a trained Remora model. Megalodon produces the training data by linking the mapped reference bases to the raw nanopore signal and annotating any potentially modified bases. 

The `merge_mappedsignalfile` command... merges the mapped signal files (nothing more). The `remora dataset merge` command could be used as well. 

The `remora dataset prepare` function takes chunks of sequence and signal around a potentially modified base and stores them in a more convenient format for training. The chunk extraction here is performed exactly as it is at inference time.

**Comment by Yijun-Tian** [comment]
Hi @marcus1487,
For the positive training sample, does it matter if I use a DNA sample with all C presented as mC? I would like to know if 100% 5mC in the positive sample cause trouble in the remora training processes. 
Thanks!
